multipy/runtime/interpreter/interpreter_impl.cpp

```diff
@@ -419,3 +436,24 @@ torch::deploy::InterpreterImpl*
 newInterpreterImpl(const std::vector<std::string>& extra_python_paths) {
   return new ConcreteInterpreterImpl(extra_python_paths);
 }
+
+extern "C" __attribute__((visibility("default")))
+torch::deploy::InterpreterImpl*
+newInterpreterImplSplit(const std::vector<std::string>& extra_python_paths)
```

Comment:
I'm a bit confused as to why this exists. It seems like the only difference between this and ConcreteInterpreterImpl is that it pulls the `_save_storages `, `_load_storages`, `_get_package `, and `_deploy_objects` functions from torch instead of multipy which we don't' want to do as we want to decouple. We should be able to just use `ConcreteInterpreterImpl`. What is the purpose of having two different versions of newInterpreterImpl?